Classes/Backend/ItemsProcFunc.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Slub\SlubEntityfacts\Backend;
+
+/***
+ * This file is part of the "slubentityfacts" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ * 
+ *  (c) 2021 Tobias Kreße <Tobias.Kresse@slub-dresden.de>, SLUB Dresden
+ * 
+ ***/
+
+class  ItemsProcFunc
+{
+  /**
+  * populating the option for links
+  *
+  * @param array &$config configuration array
+  */
+  public function user_sameAsSelected(array &$config)
+  {
+    $search = $config['row']['settings.entityfacts.personality'];
+
+    //api commmunication and decoding
+    $apiAnswer = file_get_contents('http://hub.culturegraph.org/entityfacts/'.$search);
```

Comment:
We willl keep this in this pull request. In a next step, you should make this URL configurable. E.g. by TypoScript and/or Flexform.

Always make spaces before and after the string concanating operator `'.'`. See https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/CodingGuidelines/CglPhp/PhpFileFormatting/PhpSyntaxFormatting.html#strings